.github/settings.yml

```
@@ -0,0 +1,46 @@
+repository:
+  name: holo-envoy
+  description: Server that facilitates communication between hosted Agents and a Host's Holochain Conductor
+  topics: holo, service-logging
+  private: false
+  has_issues: true
+  has_projects: false
+  has_wiki: false
+  default_branch: master
```

```suggestion
  default_branch: develop
```
Set `develop` as default to indicate where PRs should be based.